src/components/ApplicationGroup/Details/TriggerView/BulkCDTrigger.tsx

```diff
@@ -163,7 +168,8 @@ export default function BulkCDTrigger({
         if (isLoading) {
             return <Progressing pageLoader />
         }
-        const _material = appList.find((app) => app.appId === selectedApp.appId)?.material || []
+        const _currentApp = appList.find((app) => app.appId === selectedApp.appId)
+        const _material = _currentApp?.material || []
```

Comment:
Can we directly use this as it is used in only 1 place?

Reply:
It's used at multiple places down below in the code.

Reply:
Have moved _material variable though

Reply:
Done